src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -70,24 +70,28 @@ func Init() error {
 }
 
 func Configure(addr string, cloudConfigCA string, instanceId string,
-	version string, revisionDate string) {
+	version string, revisionDate string) (success chan bool) {
+	success = make(chan bool, 1)
+
 	if logglyToken == "" {
 		log.Debugf("No logglyToken, not sending error logs to Loggly")
 		return
 	}
 
 	if version == "" {
-		log.Error("No version configured, Loggly won't include version information")
+		log.Error("No version configured, not sending error logs to Loggly")
 		return
 	}
 
 	if revisionDate == "" {
-		log.Error("No build date configured, Loggly won't include build date information")
+		log.Error("No build date configured, not sending error logs to Loggly")
```

Comment:
Yes!! This mistake could have caused a lot of pain in the future, I'm sorry. Thank you for catching it on time!!